PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+## Description
+<!--- Describe your changes in detail -->
+
+<!-- For new features, provide link to FS, dev ML discussion etc. -->
+<!-- In case of bug fix, the expected and actual behaviours, steps to reproduce. -->
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
```

Comment:
I think we can have "improvement" and/or "enhancement" here too, because there could be a case which the PR falls on neither of above because of the two mentioned states.

Reply:
Also, what about code cleanup and PRs, which only add test cases?

Reply:
Correct! Those two make sense as well.